CONTRIBUTING.md

```diff
@@ -17,6 +17,10 @@ Contributions to cuDF fall into the following three categories.
     Follow the [code contributions](#code-contributions) guide below. If you
     need more context on a particular issue, please ask in a comment.
 
+As contributors and maintainers to this project,
```

Comment:
The wording here is borrowed from Pandas: https://github.com/pandas-dev/pandas#contributing-to-pandas-